system-test/read-rows.ts

```diff
@@ -123,97 +145,96 @@ describe('Bigtable/Table', () => {
     });
   });
 
-  describe('createReadStream', () => {
```

Comment:
This test used to mock out the `request` function and send responses back from the mocked request function. This was enough to test retry behaviour before because the custom retry logic was above this layer. Now that the retry logic is in gax however, these tests are replaced with tests that send the response back from a mock server instead.